portal.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
```

Comment:
Should this not belong to a System or at least a Domain?

Reply:
@benlangfeld how so? I'm not familiar with that. Suggestions, @ryanmrodriguez ?

Reply:
System I would argue against. It's a package used in building other Systems. Same logic for cobra_commander. Domain I am cool with, but we need to know the Domain to place items like these. Tech Foundations & Tools? @benlangfeld

Reply:
> Tech Foundations & Tools?

Yes

Reply:
@benlangfeld You're right about the two components. Not sure how I feel about having them front as one System though. What do you think about 2 root-level Components? They are separate entities entirely correct @xjunior ? One may be used without the other? Though it might be a bit weird, as the branding is "Power Linting", hinting at an all-in-one solution.

If we want them to be represented by a parent, maybe just have one root-level Component (power-linting), and two sub-components (rubocop-powerhome, rubocop-cobra)?

If want a System here, I need to rethink the definition.

Reply:
Does that address your concern of visibility adequately? I'd be fine with it. Where would we store said system?